Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
-    highline (2.0.0)
+    highline (2.0.3)
     method_source (0.9.2)
     mustermann (1.0.3)
     pg (0.20.0)
     pry (0.12.2)
       coderay (~> 1.1.0)
       method_source (~> 0.9.0)
-    pry-byebug (3.6.0)
-      byebug (~> 10.0)
+    pry-byebug (3.7.0)
+      byebug (~> 11.0)
       pry (~> 0.10)
-    rack (2.0.6)
-    rack-protection (2.0.4)
+    rack (2.0.8)
+    rack-protection (2.0.7)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rake (12.3.1)
-    rspec (3.8.0)
-      rspec-core (~> 3.8.0)
-      rspec-expectations (~> 3.8.0)
-      rspec-mocks (~> 3.8.0)
-    rspec-core (3.8.0)
-      rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.2)
+    rake (13.0.1)
```

Comment:
### [rake](https://github.com/ruby/rake)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 12.3.1 → 13.0.1

[_[change-log](https://github.com/ruby/rake/blob/v13.0.1/History.rdoc), [source-code](https://github.com/ruby/rake/tree/v13.0.1)_]





<details>
<summary>Commits</summary>

A change of **119** commits. See the full changes on [the compare page](https://github.com/ruby/rake/compare/v12.3.1...v13.0.1).

These are the first 10 commits:
- (acae4a2) [Fix JRuby detection on JRuby 9.2 in cpu_counter.rb](https://github.com/ruby/rake/commit/acae4a2b696d9410c428da735ae6d3364530fd76)
- (c376a93) [Use simpler RUBY_PLATFORM check for java in cpu_counter.rb](https://github.com/ruby/rake/commit/c376a932f93ac0f2dcac66002df71478a315ba42)
- (bdc6406) [Add alias `prereqs`.](https://github.com/ruby/rake/commit/bdc6406a56432a16d65aca9bf6ce82defd9718d2)
- (449766f) [Add order only dependency.](https://github.com/ruby/rake/commit/449766fb9abbfb9d8365ff824952cd1aa9067683)
- (124a03b) [Merge pull request #268 from take-cheeze/prereqs_alias](https://github.com/ruby/rake/commit/124a03bf4c0db41cd80a41394a9e7c6426e44784)
- (714a180) [Fixed bug: Task raises previous exception on second invokati…](https://github.com/ruby/rake/commit/714a18093b38661508737a6849fc7168f28829a1)
- (282b0d3) [Applied requested changes of @yuki24: Chose clean git blame …](https://github.com/ruby/rake/commit/282b0d31586c7b723f6ba7d5103f874adec1bdb9)
- (ddceed9) [Merge pull request #265 from jeremyevans/jruby92-fix](https://github.com/ruby/rake/commit/ddceed91a45b1f9dfaf1c3385f0ed4aaf7a53ecf)
- (110cc42) [remove trailing extension name in require](https://github.com/ruby/rake/commit/110cc421bb2b751fb8da24bb050040758bef9db0)
- (f34e2d5) [Fix a typo in lib/rake/application.rb](https://github.com/ruby/rake/commit/f34e2d57f01938eb1b9334b4a9354eab9e585b35)


</details>